### PR TITLE
Add extra check for empty file input in version chooser manual upload

### PR DESCRIPTION
**Pull request by JoaoMario109**

### CODE REVIEW

**Review by patrickelectric — COMMENTED** [review]
Can we move alway from v-form ?
Reason: https://github.com/bluerobotics/BlueOS/pull/2351

File: core/frontend/src/components/version-chooser/VersionChooser.vue [hunk 1]
```diff
@@ -596,6 +602,28 @@ export default Vue.extend({
...
       return this.available_versions.local.some((image) => image.sha === sha)
     },
+    get_input_file(): File | null {
+      const { files } = document.getElementById('file') as HTMLInputElement
+
+      if (files !== null && files.length > 0) {
+        return files[0]
+      }
+
+      return null
```
Comment (joaoantoniocardoso) [inline core/frontend/src/components/version-chooser/VersionChooser.vue:612]:
```suggestion
      if (files == null || files.length <= 0) {
          return null
      }
      
      return files[0]
```

just so the main/intended flow is obvious.

Comment (JoaoMario109) [reply core/frontend/src/components/version-chooser/VersionChooser.vue]: Changed to the suggested!

**Review by joaoantoniocardoso — APPROVED** [review]

File: core/frontend/src/components/version-chooser/VersionChooser.vue [hunk 2]
```diff
@@ -596,6 +602,28 @@ export default Vue.extend({
...
       return this.available_versions.local.some((image) => image.sha === sha)
     },
+    get_input_file(): File | null {
+      const { files } = document.getElementById('file') as HTMLInputElement
+
+      if (files == null || files.length <= 0) {
+        return null
+      }
+
+      return files[0]
```
Comment (patrickelectric) [inline core/frontend/src/components/version-chooser/VersionChooser.vue:612]:
```suggestion
      return files?.[0]
```

**Review by patrickelectric — CHANGES_REQUESTED** [review]

File: core/frontend/src/components/version-chooser/VersionChooser.vue [hunk 3]
```diff
@@ -596,6 +602,28 @@ export default Vue.extend({
...
+    },
+    is_file_input_not_empty(v: File | null): true | string {
+      this.file_input_error = ''
+      return !!v || this.input_file_required_message
+    },
+    validateInputFileForm(): boolean {
```
Comment (patrickelectric) [inline core/frontend/src/components/version-chooser/VersionChooser.vue:618]: The style should follow the others functions as well

**Review by patrickelectric — CHANGES_REQUESTED** [review]

File: core/frontend/src/components/version-chooser/VersionChooser.vue (hunk 3 above)
Comment (JoaoMario109) [reply core/frontend/src/components/version-chooser/VersionChooser.vue:614]: Changed both the methods and the computed property to follow the convention, only error messages remained snake case due the others data be in this way also

**Review by patrickelectric — APPROVED** [review]

**Pull request merged**